questions/Gallery/AstOfCProgram/solution.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <iomanip>
+using namespace std;
+
+int main() {
+    const double DISCOUNT = 1.5e-1;
+    double price;
+    
+    cout << "Enter an item price: $";
+    cin >> price;
+    cout << fixed << setprecision(2) << price * (1.0 - DISCOUNT) << endl;
```

Comment:
The example includes a text, the solution should probably include it too, even if it's not enforced in the autograder.
```suggestion
    cout << "The sale price is " << fixed << setprecision(2) << price * (1.0 - DISCOUNT) << endl;
```